examples/capture/main.rs

```diff
@@ -27,20 +29,37 @@ async fn run() {
         .await
         .unwrap();
 
-    // Rendered image is 256×256 with 32-bit RGBA color
-    let size = 256u32;
+    let args: Vec<_> = env::args().collect();
+    let (width, height) = match args.len() {
+        // 0 on wasm, 1 on desktop
+        0 | 1 => (100usize, 200usize),
+        3 => (args[1].parse().unwrap(), args[2].parse().unwrap()),
+        _ => {
+            println!("Incorrect number of arguments, possible usages:");
+            println!("*   0 arguments - uses default width and height of (100, 200)");
+            println!("*   2 arguments - uses specified width and height values");
+            return;
+        }
+    };
+
+    // It is a webgpu requirement that BufferCopyView.layout.bytes_per_row % 256 == 0
+    // So we calculate an appropriate padded_bytes_per_row that fits that requirement
+    let bytes_per_pixel = size_of::<u32>();
+    let real_bytes_per_row = width * bytes_per_pixel;
```

Comment:
`unpadded_bytes_per_row` is probably a better name for this as `real` is one of those annoying words that could mean either the padded or the unpadded size.

Reply:
good idea